api/serializers.py

```diff
@@ -27,14 +27,24 @@ def save(self, **kwargs):
 
 class CounterSerializer(serializers.HyperlinkedModelSerializer, ReadOnlySerializer):
     geometry = serializers.SerializerMethodField()
+    properties = serializers.SerializerMethodField()
+    type = serializers.CharField(default="Feature")
 
     class Meta:
         model = Counter
-        fields = ["id", "name", "classifying", "crs_epsg", "source", "geometry"]
+        fields = ["type", "id", "geometry", "properties"]
 
     def get_geometry(self, obj):
         return {"type": "Point", "coordinates": [obj.geom.x, obj.geom.y]}
 
+    def get_properties(self, obj):
+        return {
+            "id": obj.id,
+            "name": obj.name,
```

Comment:
How about a case where name does not exist, would it be a case to handle? Is it possible for it to not exist